habitat_baselines/rl/ddppo/policy/resnet_policy.py

```diff
@@ -99,7 +97,6 @@ def from_config(
             rnn_type=config.RL.DDPPO.rnn_type,
             num_recurrent_layers=config.RL.DDPPO.num_recurrent_layers,
             backbone=config.RL.DDPPO.backbone,
-            normalize_visual_inputs="rgb" in observation_space.spaces,
```

Comment:
@ASzot @erikwijmans My change was to set normalize_visual_inputs to `True` all the time, which is different from what is happening here. I think the current behavior is that only the visual goal images are normalized, the new one would be all observations are normalized. Is that reasonable? Should I keep normalize_visual_inputs the way it is?

Reply:
I don't know what the performance of this normalization will be with depth only input (all rearrange tasks are by default depth only). Can you possibly run a quick Pick task to make sure this does not degrade performance? Conceptually, I think this is fine and am okay with it.

Reply:
This is the curve for the learning of ddppo_pick with the change. Does it look correct to you?
![test_ddppo_pick log](https://user-images.githubusercontent.com/28320361/187251385-0e7baae3-2782-4a8a-83f0-3cefe8b568fd.png)

Reply:
And this is the comparison before and after the change : 
![test_ddppo_pick log_dev](https://user-images.githubusercontent.com/28320361/187731190-fdc13cc4-af98-4d18-b2e1-4959c41f1e08.png)
I see no noticeable change.